src/bsp/platform/iotc_bsp_io_net_test.cc

```diff
@@ -0,0 +1,129 @@
+/* Copyright 2018-2019 Google LLC
+ *
+ * This is part of the Google Cloud IoT Device SDK for Embedded C,
+ * it is licensed under the BSD 3-Clause license; you may not use this file
+ * except in compliance with the License.
+ *
+ * You may obtain a copy of the License at:
+ *  https://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "../../tests/tools/iotc_test_echoserver.h"
+#include "gtest.h"
+#include "iotc_bsp_io_net.h"
+// TODO(sungju): get rid of relative path for inclusion
```

Comment:
ping: todo

Reply:
Thanks for reminding!  I made a bug for this. (b/127243999)

Reply:
b/127243999 is now closed